<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuWindow</class>
 <widget class="QMainWindow" name="MainMenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="player1select">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="player2select">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="p1display">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="p2display">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleDisplay">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Othello</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
